cypress/e2e/web/login.js

```diff
@@ -10,7 +10,8 @@ describe('user', () => {
 		cy.get('#password').should('not.exist');
 
 		cy.findByText(/^(entrar|sign in)$/i).should('not.exist');
-		cy.get('div[class*=LoginBox] button[type=button]').click();
+		// cy.get('div[class*=LoginBox] button[class*=UserButton]').click();
+		cy.visit('/user/my-account');
```

Comment:
Anteriormente, no momento do logout, funcionava assim: clicava na caixa de login, que levava para a página de meu perfil do usuário e, em seguida, clicava no botão de login.

Porém, agora, ao clicar na caixa de login, é aberto um dropdown. Logo, o **ideal** seria clicar no botão de logout que há dentro desse dropdown. Entretanto, como o dropdown tem um position absolute, o cypress não enxerga o conteúdo dentro dele.

Teste alguns workarounds, como 
```
 		cy.get('button[class*=LogoutButton]').click({force: true}); 
```
Mas não funcionou. Logo, modifiquei para que visite a página de perfil do usuário e pegue o botão de logout de lá da área de meu perfil do usuário.

Alguma sugestão para que possa testar aqui?

Reply:
https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Visibility

Provavelmente o cypress está considerando todo o UserDropdown como elemento não visível porquê o DropdownContainer tem `height` 0.

![image](https://user-images.githubusercontent.com/6104632/87881915-fccf9880-c9d2-11ea-895c-e3483142fedf.png)

Reply:
Fiz vários testes aqui. É necessário setar o `height` e o `width`. Tem uma seçaõ no mesmo link que você enviou que mostra [casos adicionais em que um elemento é considerado hidden](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Additionally-an-element-is-considered-hidden-if). 

No nosso caso, temos um Header (sticky) -> LoginBox -> DropDownContainer (relative) -> DropDownMenu (relative). Tem várias características do nosso layout que podem se encaixar em uma dessas situações. Eu até consegui fazer com que o cypress reconheça o menu, mas sempre quebrando nosso layout ou, então, deixando o layout engessado, o que não faria sentido em prol de fazer o cypress passar pelo DropDown. Pelo que percebi, eu tenho de colocar o `DropDownContainer` (relative) com uma largua mínima maior ou igual ao `DropDownMenu` (absolute),  o que acaba quebrando o layout:
![image](https://user-images.githubusercontent.com/12154623/87938123-6ea7f080-ca6c-11ea-9dad-f8023d5ec8b5.png)
Daí o teste passa:
![image](https://user-images.githubusercontent.com/12154623/87939041-d3b01600-ca6d-11ea-92d1-3cc0c36eabb5.png)


Considerando o tempo, que o objetivo de testar é verificar se o usuário consegue logar e deslogar e o botão de logout do meu perfil é o mesmo utilizado no dropdown, sugiro continuarmos deixando o teste indo através da URL e usando o botão de logout do Meu Perfil.

Reply:
Vamos criar uma outra issue (low-priority) pra ver isso daqui, tenho a impressão de que tem como fazer consertar issso aqui.

Reply:
Blz :+1: .